bitcoind-rpc/src/main/scala/org/bitcoins/rpc/serializers/JsonReaders.scala

```diff
@@ -529,10 +529,10 @@ object JsonReaders {
   }
 
   // Currently takes in BTC/kB
-  implicit object BitcoinFeeUnitReads extends Reads[BitcoinFeeUnit] {
-    override def reads(json: JsValue): JsResult[BitcoinFeeUnit] =
-      SerializerUtil.processJsNumber[BitcoinFeeUnit](num =>
-        SatoshisPerByte(Satoshis((num * 100000).toBigInt)))(json)
+  implicit object BitcoinFeeUnitReads extends Reads[BitcoinFeeRate] {
```

Comment:
Didn't we decide to change this type to `Reads[BTCPerKiloByte]` so we don't use the wrong implicit? 

https://bitcoin.org/en/developer-reference#estimatesmartfee

Also it is concerning that unit tests aren't failing when we use the wrong implicit here?